### DISCUSSION TITLE
expense-ansible : Getting error while downloading content and extracting in frontend server.

### DISCUSSION THREAD

**Discussion by sindhu9227** [learndevopsonline/learndevopsonline#1897] [Q&A] [answered]
Session 15 - frontend ansible : While downloading content and extracting, I am not sure about this error. I have created a new server before running this command ( ansible-playbook -i 54.167.89.74, -e ansible_user=ec2-user -e ansible_password=DevOps321 frontend.yml )

Please unblock me to proceed further.

![Screenshot 2024-05-16 170306](https://github.com/learndevopsonline/learndevopsonline/assets/169046117/6d559a8e-1a92-48e6-b61d-5e92c6a637fe)

**Comment by chaithanya1812** [comment, marked as answer]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/f86761c2-5762-471d-b1cd-10cbcafdc06e)

**Reply by sindhu9227** [reply]
Thanks! It worked
